client/.env

```diff
@@ -1,14 +1,14 @@
 PORT=1729
-REACT_APP_HTTP_BACKEND_URL=http://localhost:1000
+REACT_APP_HTTP_BACKEND_URL=https://staging.arkhn.com/pyrog-api
 REACT_APP_RIVER_URL=https://staging.arkhn.com/river
 REACT_APP_PAGAI_URL=https://staging.arkhn.com/pagai
 REACT_APP_CLEANING_SCRIPTS_URL=https://staging.arkhn.com/scripts
 REACT_APP_FHIR_API_URL=https://staging.arkhn.com/api
 REACT_APP_CLIENT_ID=local-pyrog-client
 REACT_APP_CLIENT_SECRET=pyrog-client
-REACT_APP_AUTH_URL=http://localhost:4444/oauth2/auth
-REACT_APP_TOKEN_URL=http://localhost:4444/oauth2/token
-REACT_APP_REVOKE_URL=http://localhost:4444/oauth2/revoke
-REACT_APP_LOGOUT_URL=http://localhost:4444/oauth2/sessions/logout
+REACT_APP_AUTH_URL=https://staging.arkhn.com/hydra/oauth2/auth
+REACT_APP_TOKEN_URL=https://staging.arkhn.com/hydra/oauth2/token
+REACT_APP_REVOKE_URL=https://staging.arkhn.com/hydra/oauth2/revoke
+REACT_APP_LOGOUT_URL=https://staging.arkhn.com/hydra/oauth2/sessions/logout
```

Comment:
Are you sure we want to commit this ?

Reply:
no we should keep local urls, but the .env should be in the gitignore an not be comitted anyway.